addons/hr_timesheet/models/res_company.py

```diff
@@ -36,14 +36,9 @@ def _default_timesheet_encode_uom_id(self):
     timesheet_encode_uom_id = fields.Many2one('uom.uom', string="Timesheet Encoding Unit",
         default=_default_timesheet_encode_uom_id, domain=lambda self: [('category_id', '=', self.env.ref('uom.uom_categ_wtime').id)])
     internal_project_id = fields.Many2one(
-        'project.project', string="Internal Project",
+        'project.project', string="Internal Project", inverse="_inverse_is_internal_project",
```

Comment:
If we keep the compute for `is_internal_project` in project.project, this won't be needed. If we do keep it, I think a name like `_inverse_internal_project_id` would fit better